fuzzers/frida_libpng/harness_win.cpp

```diff
@@ -7,8 +7,12 @@ extern "C" __declspec(dllexport) size_t
     LLVMFuzzerTestOneInput(const char *data, unsigned int len) {
   if (data[0] == 'b') {
     if (data[1] == 'a') {
-      if (data[2] == 'd') {}
+      if (data[2] == 'd') {
+        // STATUS_ACCESS_VIOLATION
+        int *p = 0x0;
```

Comment:
This still needs to be volatile (i.e., `int volatile *p = 0x0;`), see:
<img width="1347" alt="image" src="https://user-images.githubusercontent.com/297744/170956559-30e0a61e-ac66-485b-b900-260fc4b4188a.png">